_sections/10-structure.md

```diff
@@ -27,18 +26,19 @@ Below is an example of a complete and valid ODK XForm:
         <h:title>My Survey</h:title>
         <model>
             <instance>
-                <data id="mysurvey" version="2014083101">
+                <data id="mysurvey" orx:version="2014083101">
```

Comment:
Yes, that's excellent. I've added a [separate issue](https://github.com/opendatakit/odk-xform-spec/issues/55) for this, so this PR stays focused on #33.